sidebars.js

```diff
@@ -8,154 +8,179 @@
   "docs": [
     {
-      "Welcome": [
+      "Get started": [
       "overview",
-      "saas-vs-self-hosted",
+      "self-hosted/install/preparation",
       {
         "type": "category",
-        "label": "Getting Started",
+        "label": "Installing Okteto",
         "items": [
-          "getting-started",
-          "using-dev-envs"
+          "self-hosted/install/deployment",
+          {
+            "type": "category",
+            "label": "Cloud provider guides",
+            "items": [
+              "self-hosted/eks",
+              "self-hosted/aks",
+              "self-hosted/civo",
+              "self-hosted/do",
+              "self-hosted/gke"
+            ]
+          },
         ]
       }
     ],
   },
   {
-    "Dev Environments": [
-      "reference/development-environments",
+    "Core concepts": [
       "cloud/okteto-cli",
       {
         "type": "category",
-        "label": "Deploying",
+        "label": "Okteto components",
         "items": [
-          "cloud/launch-from-catalog",
-          "cloud/deploy-from-git",
-          "cloud/private-repositories",
-        ]
+          "cloud/namespaces",
+          "cloud/build",
+          "cloud/registry",
+          "cloud/use-volume-snapshots",
+        ],
       },
-      "cloud/external-resources",
-      "cloud/secrets",
       {
         "type": "category",
-        "label": "Exposing Your Application",
+        "label": "Ingress",
         "items": [
           "cloud/ssl",
           "cloud/private-endpoints",
-          "cloud/custom-domains"
-        ]
+        ],
       },
-      "cloud/build",
-      "cloud/registry",
-      "cloud/use-volume-snapshots",
-      "cloud/namespaces",
-      "cloud/credentials",
-      "cloud/personal-access-tokens",
-      "reference/file-synchronization",
-      "reference/ssh-server"
-    ],
+      {
+        "type": "category",
+        "label": "Credentials",
+        "items": [
+          "cloud/credentials",
+          "cloud/personal-access-tokens",
+        ],
+      },
+    ]
   },
   {
-    "Preview Environments": [
-      "cloud/preview-environments/preview-environments",
+    "Guides": [
       {
         "type": "category",
-        "label": "For GitHub",
+        "label": "Tutorials",
         "items": [
-          "cloud/preview-environments/dashboard",
-          "cloud/preview-environments/preview-environments-github"
-        ]
+          "tutorials/compose-getting-started",
+          "tutorials/webpack",
+          "tutorials/aws-lambda"
+        ],
       },
-      "cloud/preview-environments/preview-environments-gitlab"
+      {
+        "type": "category",
+        "label": "Samples",
+        "items": [
+          "samples/aspnetcore",
+          "samples/golang",
+          "samples/java",
+          "samples/node",
+          "samples/php",
+          "samples/python",
+          "samples/ruby",
+          "samples/more"
+        ],
+      },
+    ]
+  },
+  {
+    "Okteto manifest": [
```

Comment:
While this section doesn't grow, should we remove it and move the 2 docs under `core concepts`?

Reply:
The most important page missing (imo) is the "Configure the manifest", which we don't yet have content for and would come in a later PR.

I think it's worth keeping this at the top-level and prioritizing the content for that missing page since I do think it makes sense why we moved this to the top-level

Reply:
SGTM